requirements/partial/requirements_development.txt

```diff
@@ -11,6 +11,7 @@ pytest-cov==5.0.0
 pytest-profiling==1.7.0
 pyupgrade==3.15.2
 pyyaml==6.0.1
+psycopg[binary]==3.1.18  # only dev, in production use psycopg
```

Comment:
May be should move this to the production requirement file, which should be used for all the common base packages.